File: AppLoginJavaApp/app/src/main/java/com/paulo/apploginjavaapp/MainActivity.java
package com.paulo.apploginjavaapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
Button btnEntrar;
EditText edtUsuario;
EditText edtSenha;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnEntrar = findViewById(R.id.btnEntrar);
        edtUsuario = findViewById(R.id.edtUsuario);
        edtSenha = findViewById(R.id.edtSenha);

        btnEntrar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String usuario = edtUsuario.getText().toString();
                String senha = edtSenha.getText().toString();
                if(usuario.equals("user") && senha.equals("123")){
                    Toast.makeText(getApplicationContext(),"Senha aceita", Toast.LENGTH_LONG).show();

                }else{
                    Toast.makeText(getApplicationContext(),"Senha ERRADA", Toast.LENGTH_LONG).show();
                }
                edtUsuario.setText("");
                edtSenha.setText("");
            }
        });

    }
}

File: LoginApp/app/src/main/java/com/paulo/loginapp/MainActivity.kt
package com.paulo.loginapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import com.paulo.loginapp.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding;
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnEntrar.setOnClickListener {
            val username = binding.edtUser.text.toString().trim();
            val pasword = binding.edtSenha.text.toString().trim();

            if(username.equals("user") && pasword.equals("123456"))
            {
                Toast.makeText(applicationContext, "Login Ok", Toast.LENGTH_LONG).show();

                var i = Intent(this, LoginCertoActivity::class.java)
                startActivity(i);



            }else{
                var i = Intent(this, LoginErradoActivity::class.java)
                startActivity(i);
                Toast.makeText(applicationContext, "Login ou senha Inválido", Toast.LENGTH_LONG).show();

            }

            binding.edtUser.setText("");
            binding.edtSenha.setText("");
        }

        binding.btnFechar.setOnClickListener {

            Toast.makeText(applicationContext,"VOLTE SEMPRE", Toast.LENGTH_LONG)
            finish();

        }

    }
}